qutebrowser/config/configdata.yml

```diff
@@ -1291,10 +1291,22 @@ statusbar.widgets:
         - tabs: "Current active tab, e.g. `2`."
         - keypress: "Display pressed keys when composing a vi command."
         - progress: "Progress bar for the current page loading."
+        - settings: "Boolean settings state for the current page."
     none_ok: true
   default: ['keypress', 'url', 'scroll', 'history', 'tabs', 'progress']
   desc: List of widgets displayed in the statusbar.
 
+settings.widget.content:
+  type:
+    name: List
+    valtype:
+      name: String
+    none_ok: true
+  default: []
+  desc: Which boolean settings to display in the statusbar settings widget.
```

Comment:
Other multiline-description are opened by `desc: >-` and continued on the next line. I'd suggest to do the same here for consistency.